### DISCUSSION TITLE
Review Requester

### DISCUSSION THREAD

**Discussion by duncandewhurst** [OP-TED/eForms-SDK#258] [Q&A] [answered]
The 'Review Requester' role appears in [Table 25. Roles marked using XML elements](https://docs.ted.europa.eu/eforms/latest/schema/all-in-one.html#rolesMarkedUsingXmlElementsTable) associated with the following XPath: `/ContractAwardNotice​/ext:UBLExtensions​/ext:UBLExtension​/ext:ExtensionContent​/efext:EformsExtension​/efac:AppealsInformation​/efac:AppealStatus​/efac:AppealingParty`. However, I can find no other reference to the `efac:AppealsInformation` element or its children in the SDK documentation nor in the [examples](https://github.com/OP-TED/eForms-SDK/tree/develop/examples).

Can `efac:AppealingParty` actually appear in a notice XML file? Are there any other children of `efac:AppealsInformation` that do not appear in the SDK documentation, but can appear in a notice XML file?

**Comment by YvesJo** [comment, marked as answer]
This role is not yet to be considered. It is exclusively for E5 notices which will only be considered in the future. Further information will be provided later when time will come.

**Reply by duncandewhurst** [reply]
Thanks! We'll disregard this role for now.
